docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+| `changeState` | string | Type of state change used for the virtual page navigation | `pushState`, `replaceState` | Recommended |
+
+**[1]:**  Alias for [`http.url`](../../../trace/semantic_conventions/http.md)
+**[2]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page - Initial page load within the browser and will generally also precede a PageNavigationTiming event.|
+| `1` | virtual_page - This is for Single Page Applications (SPA) where the framework provides the ability to perform client side only page "navigation", the exact definition of what a virtual page change is determined by the SPA and the framework it is using.|
+
+## PageNavigationTiming
+
+The event name MUST be`page_navigation_timing`.
+
+This event describes the timing metrics of a page navigation as provided by
+`PerformanceNavigationTiming` Performance API.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| name | string | URL of the page || Recommended |
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+## ResourceTiming
+
+The event name MUST be `resource_timing`.
+
+This event describes the timing metrics as provided by `PerformanceResourceTiming`
+Performance API.  These metrics are related to fetching a resource, such as
+XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+|name | string | URL of the requested resource || Recommended |
+|fetchStart | long | || Recommended |
+|domainLookupStart | long | || Recommended |
+|domainLookupEnd | long | || Recommended |
+|connectStart | long | || Recommended |
+|secureConnectionStart | long | || Recommended |
+|connectEnd | long | || Recommended |
+|requestStart | long | || Recommended |
+|responseStart | long | || Recommended |
+|responseEnd | long | || Recommended |
+
+## Exception
+
+The event name MUST be `browser.exception`.
```

Comment:
any reason to introduce yet another way to record exceptions? [span events](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md) - stable, [log events](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-logs.md) - stable, now browser events.

Can we find a way to consolidate them?

since log events are already stable, we should try to express browser events using them.

Reply:
1. Span event with name `exception` is for recording exceptions during a span
2. Log records can have `exception.*` attributes recorded by Log Bridge API when translating exceptions in application logs to OpenTelemetry log records. Note that they are still not events, that is, they will not have `event.name` attribute.
3. Hence, to record exceptions as independent events we still need a separate semantic convention. In the Event SIG, it was suggested to have just one event with name `exception` without any domain prefix and have the `exception.*` as fields in the payload of this event. I will create a separate PR for this.

Reply:
The `exception` event is moved to a separate PR - https://github.com/open-telemetry/semantic-conventions/pull/970